scripts/retrieve_block/retrieve_block.go

```diff
@@ -113,15 +113,17 @@ func main() {
 
 	protocolID := protocol.ID(fmt.Sprintf("/%s/sync/2", chain.ProtocolID))
 	for _, bootnodesAddr := range bootnodes {
+		log.Println("connecting...")
 		err := p2pHost.Connect(ctx, bootnodesAddr)
 		if err != nil {
+			fmt.Printf("fail with: %s\n", err.Error())
 			continue
 		}
 
 		log.Printf("requesting from peer %s\n", bootnodesAddr.String())
 		stream, err := p2pHost.NewStream(ctx, bootnodesAddr.ID, protocolID)
 		if err != nil {
-			log.Printf("WARN: failed to create stream using protocol %s: %s", protocolID, err.Error())
+			fmt.Printf("WARN: failed to create stream using protocol %s: %s", protocolID, err.Error())
```

Comment:
I'm wondering why these messages are being printed instead of being logged?

Reply:
fixed, now using `log.Printf.`